Makefile

```diff
@@ -29,6 +29,7 @@ lint-dlt-init:
 
 lint-code:
 	./check-package.sh
+	./check-requirements.py
```

Comment:
I'd move `./check-requirements.py` to `lint-dlt-init`. as a consequence it will run this check in `init` CI workflow which has dev dependencies.